TicketBooking.Data/DataModel/ApplicationUser.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicketBooking.Data.DataModel
+{
+    public class ApplicationUser : IdentityUser
+    {
```

Comment:
what is the relationship between `ApplicationUser` with other table? Is Passenger should have relationship with this?

Reply:
ApplicationUser is linked with REPORT table and only for the users who already signed up and require to access methods that need to log in.
Passenger does not need to have relationship with ApplicationUser since even without log in, users are still able to book tickets, and one user can fill the information of multiple passengers.

Reply:
what is the Report table used for? 
If I'm a user, I booking some tickets, then How can I see the history?

Reply:
1st question: Report table is used to store the flight info. 
2nd question: it is divided into 2 cases:
- log in -> book ticket -> can view the history from Report table
- not log in -> book ticket -> cannot view the history from Report table (but in case users want to check his old flights, as long as he has the flight ID then it is possible)